### PR TITLE
Fix for version 0.6

### PR DESCRIPTION
**Pull request by GeorgeEngland**
We need this change for version 0.6

### CODE REVIEW

**Comment by moriarty** [comment]
This issue is discussed in #681 
GitHub reverted the change upstream, so no change should be needed but it's trickling through different content delivery networks and ISP caches etc.
https://twitter.com/tgummerer/status/1488493440103030787

**Comment by chases2** [comment]
From the discussion on https://github.com/bazel-contrib/SIG-rules-authors/issues/11#issuecomment-1026769107, it seems like referencing the `/refs/tags` endpoint is still preferable.

File: k8s/k8s.bzl [hunk 1]
```diff
@@ -49,7 +49,7 @@ py_library(
 """,
             sha256 = "e9df8412ddabc9c21b4437ee138875b95ebb32c25f07f962439e16005152e00e",
             strip_prefix = "pyyaml-5.1.2",
-            urls = ["https://github.com/yaml/pyyaml/archive/5.1.2.zip"],
+            urls = ["https://github.com/yaml/pyyaml/archive/refs/tags/5.1.2.zip"]
```
Comment (chases2) [inline k8s/k8s.bzl:52]: Needs a trailing comma

**Comment by fejta** [comment]
Would you like to fixup this PR or close it out?

**Comment by fejta** [comment]
Please rebase and reopen if you are still interested

**Pull request closed without merging**